### DISCUSSION TITLE
Add some more values to the summary stat.tsv table

### DISCUSSION THREAD

**Discussion by KeepCalmAndRushB** [MannLabs/alphadia#154] [Ideas]
I got a stat.tsv summary table which has:

run			
precursors			
proteins			
ms1_accuracy 				
fwhm_rt		
fwhm_mobility 

Could you please add some more values?

Gradient lenght *
n MS *
n MSMS *
			
MS:
median Tic		
median It
median Cycle time

MSMS:
median Tic		
median It	
	
Precursors:
median width_rt			
median width_mobility
median Tailing (Tosoh) at 10% peak height - or median Tailing (USP) at 5% of peak height	
median Intensity			
median Score			
median Number of concurrent MSMS per precursors

* NaJa, for a fast method check

**Comment by mschwoer** [comment]
this now tracked internally
